js/bundle.go

```diff
@@ -409,9 +430,25 @@ func (b *Bundle) setInitGlobals(rt *sobek.Runtime, vu *moduleVUImpl, modSys *mod
 			return nil, errors.New("open() can't be used with an empty filename")
 		}
 		// This uses the pwd from the requireImpl
-		pwd := impl.internal.CurrentlyRequiredModule()
-		return openImpl(rt, b.filesystems["file"], &pwd, filename, args...)
+		pwd, err := modSys.CurrentlyRequiredModule()
+		if err != nil {
+			return nil, err
+		}
+		return openImpl(rt, b.filesystems["file"], pwd, filename, args...)
 	})
+	warnAboutModuleMixing := func(name string) {
+		warnFunc := rt.ToValue(func() error {
+			return fmt.Errorf(
+				"you are trying to access identifier %q, this likely is due to mixing ESM and CommonJS syntax",
```

Comment:
```suggestion
				"you are trying to access identifier %q, this likely is due to mixing ESM and CommonJS syntax." + 
				"This isn't supported in the JavaScript standard, please use only one or the other",
```

cc @sniku @grafana/k6-core